nixops_aws/resources/rds_db_subnet_group.py

```diff
@@ -26,6 +26,20 @@ def get_type(cls):
     def get_resource_type(cls):
         return "rdsSubnetGroups"
 
+    def __init__(self, name: str, config: nixops.resources.ResourceEval):
+        super(RDSDbSubnetGroupDefinition, self).__init__(name, config)
+
+        self.group_name: str = self.config.name
```

Comment:
I thought it was mostly about RDS snapshot & boto 3 port, is this change really required in this PR?

Reply:
This change is to mainly fix the destroy operation bug related to the subnet-group resource but also to streamline the resource definition and follow the standard